frontend/src/components/order/order.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  justify-content: space-around;
+  border-color: black;
+  border-width: 2px;
+  border-radius: 10px;
+  border-style: solid;
+  font-size: 1.5rem;
+}
+
+.green {
+  color: green;
+}
+
+.red {
+  color: red;
+}
+
+.th {
```

Comment:
We may want this also to be named after a particular table header since we may end up with many tables that have slightly different styling.